packages/main/src/themes/sap_horizon/Panel-parameters.css

```diff
@@ -8,4 +8,5 @@
 	--_ui5_panel_border_bottom: none;
 	--_ui5_panel_outline_offset: 0px;
 	--_ui5_panel_title_font_weight: bold;
+	--_ui5_panel_border_radius_expanded: var(--sapElement_BorderCornerRadius) var(--sapElement_BorderCornerRadius) 0 0;
```

Comment:
I think this should be e single value for the border-radius: 0, 0.25rem, 0.5rem or

Reply:
Nope, since we only want to round 2 of 4 sides.